generators/app/templates/src/scss/margins.scss

```diff
@@ -1,5 +1,6 @@
 $breakpoint-tablet: 768px;
 $breakpoint-desktop: 1024px;
+$scale: 1;
```

Comment:
isn't this too low? Maybe 2/4?

Reply:
Honestly I do not know, just put this variable because without, the app 💥

Reply:
Great!